### PR TITLE
Better layout, hardlink output file, etc.

### PR DESCRIPTION
**Pull request by oskar456**
In this PR:
 * Layout of 16:9 slides together with speaker is slightly improved
 * There's a generator of proper file names, that are dropped (as a hardlink) into the source directory (changeable by `--outpath` command option)
 * Small PEP8 fixes
 * Workaround for Speaker-only videos – otherwise ffmpeg crashed with error: `more samples than frame size (avcodec_encode_audio2)` – proper solution would be handy.

Feel free to drop/cherry-pick either commit ;)

--
Ondřej

### CODE REVIEW

File: pyvo/make_vid.py [hunk 1]
```diff
@@ -168,4 +182,18 @@ def make_pyvo(
 
     print(result.graph)
 
+    outname = [date.strftime('%Y-%m-%d'), event, speaker, title,
+               'preview' if preview else None]
+    outname = filter(lambda x: x, outname)
+    outname = [slugify(x) for x in outname]
```
Comment (encukou) [inline pyvo/make_vid.py]: Nitpick: `[slugify(x) for x in outname if x]` should be shorter and faster than pre-`filter`-ing

**Comment by encukou** [comment]
Looks good! Before merging, could you produce a widescreen video with it, to make sure it works?

File: pyvo/make_vid.py [hunk 2]
```diff
@@ -183,7 +183,7 @@ def make_pyvo(
     if praha:
         overlay = export_template.exported_slide('slide-overlay', duration=main.duration)
         main |= overlay
-    main = main.faded_out(0.5)
+    main = main.faded_out(0.5).faded_in(0.5)
```
Comment (encukou) [inline pyvo/make_vid.py]: I think the fade-in should be skipped for `speaker_only` videos

Comment (oskar456) [reply pyvo/make_vid.py]: I did some speaker-only videos and I don't think it's bad for them either. Anyway, feel free to ignore these commits until you have the time to see the results.

**Comment by encukou** [comment]
Tried it. It is great. Thank you – both for the PR and for the videos!

**Pull request merged**